### PR TITLE
docs for components-path changed to resources-path

### PR DESCRIPTION
**Pull request by pravinpushkar**
Signed-off-by: Pravin Pushkar <ppushkar@microsoft.com>

Thank you for helping make the Dapr documentation better!

## Issue reference

1st PR (--components-path changed to resources-path)- https://github.com/dapr/cli/pull/1012
2nd PR( components directory changed to resources direcotory)- https://github.com/dapr/cli/issues/1149

### CODE REVIEW

**Comment by msfussell** [comment]
This PR needs to be extend to cover more than just the CLI update. This is a fundamental change across many docs, since this will become the recommended way to use components. Therefore we need to update the following
- https://docs.dapr.io/concepts/components-concept/#component-specification
- All HowTo where we use components path in examples/sample. e.g. https://docs.dapr.io/developing-applications/building-blocks/pubsub/howto-publish-subscribe/#set-up-the-pubsub-component and https://docs.dapr.io/developing-applications/building-blocks/state-management/howto-get-save-state/#set-up-a-state-store
- There may be other places. 

I suggest that we update the Quickstarts https://github.com/dapr/quickstarts/issues/769 and @hananah can maybe help here

**Comment by pravinpushkar** [comment]
lets wait on this, until we decide on the PR - https://github.com/dapr/cli/pull/1149. Will open it for review accordingly.

**Comment by pravinpushkar** [comment]
I will open it for review shortly

**Comment by pravinpushkar** [comment]
/keep-alive -  since the feature PR is still not merged

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 1]
```diff
@@ -83,14 +83,14 @@ docker ps
 
 <img src="/images/install-dapr-selfhost/docker-containers.png" width=800>
 
-### Step 5: Verify components directory has been initialized
+### Step 5: Verify resources directory has been initialized
 
-On `dapr init`, the CLI also creates a default components folder that contains several YAML files with definitions for a state store, Pub/sub, and Zipkin. The Dapr sidecar will read these components and use:
+On `dapr init`, the CLI also creates a default resources folder that contains several YAML files with definitions for a state store, Pub/sub, and Zipkin. The Dapr sidecar will read these resources and use:
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md:88]:
```suggestion
On `dapr init`, the CLI also creates a default resources folder that contains several YAML component files with specifications for a state store, pub/sub, and Zipkin. The Dapr sidecar loads these component resources and uses:
```

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 2]
```diff
@@ -83,14 +83,14 @@ docker ps
...
 
 - The Redis container for state management and messaging.
```
Comment (msfussell) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md:90]:
```suggestion
- The Redis container for both state management and pub/sub messaging.
```

**Review by msfussell — COMMENTED** [review]
@pravinpushkar - Looks like you added this file `daprdocs/package-lock.json` by accident.

**Comment by pravinpushkar** [comment]
> @pravinpushkar - Looks like you added this file `daprdocs/package-lock.json` by accident.

Don't we need this ? Normally, we do checkin the lock file for node project AFAIK ?

**Comment by pravinpushkar** [comment]
Aborting this PR as feature PR is not merged for this release - https://github.com/dapr/cli/pull/1149.
I will open a fresh PR once that gets merged.

and doc changes for resources-path has already been covered via a different PR -  https://github.com/dapr/docs/pull/3016

**Pull request closed without merging**